backend/fms_core/utils.py

```diff
@@ -30,6 +31,12 @@
 
 TRUTH_VALUES = frozenset({"TRUE", "T", "YES", "Y"})
 
+def comma_separated_string_to_array(s):
+    """
+    Returns emtpy string if argument is a blank string or None,
+    otherwise it returns an array of strings
+    """
+    return s.replace(" ", "").split(',') if s else ""
```

Comment:
this is dangerous ... that replace may not be anticipated when we call comma_separated_string_to_array. In some condition we may want to preserve the spaces. Don't you want strip() instead ? less risky. Also, the return value is either a string or an array ... if there is no element it would be better to return either an empty list or a list with an empty string as sole element.

Reply:
strip doesn't remove white spaces in the middle of the string so "ABC ABC" would still be "ABC ABC". The replace is basically removing whitespaces, which if you want an array of strings I believe normally you would want the items of the array to be stripped.